test/cucumber/http_response.go

```diff
@@ -66,11 +66,14 @@ func init() {
 		ctx.Step(`^the response should match json:$`, s.TheResponseShouldMatchJsonDoc)
 		ctx.Step(`^the response should match:$`, s.theResponseShouldMatchText)
 		ctx.Step(`^the response should match "([^"]*)"$`, s.theResponseShouldMatchText)
+		ctx.Step(`^the response should match expression "([^"]*)"$`, s.theResponseShouldMatchExpression)
```

Comment:
are these steps actually being used anywhere?

Reply:
I think I used them at some point, should I remove them ? they are meant to act like a predicate

Reply:
removed